app-nuxt-base/pages/index.vue

```diff
@@ -0,0 +1 @@
+<template>this is a base project</template>
```

Comment:
An empty `/pages/index.vue` exists here, but once user's project includes the file under the same path, that will be used instead of this.